frontend/src/component/menu/routes.ts

```diff
@@ -246,6 +247,15 @@ export const routes: IRoute[] = [
         type: 'protected',
         menu: { mobile: true, advanced: true },
     },
+    {
+        path: '/releases-management',
+        title: 'Release management',
+        component: ReleaseManagement,
+        type: 'protected',
+        menu: { advanced: true, mode: ['enterprise'] },
+        flag: 'releasePlans',
```

Comment:
This @gardleopard makes sure it's shown only when the specified flag is 'true'
- Another interesting side effect, setting that flag to true negates any enterprise check for the link, so would show up in both OSS and Pro, but the route will still yield a 404 in that case.
Still, it's something we might want to look at (but I feel that's beyond the scope of this PR, I'm making a bug task for it though)